lib/containers/urls.pm

```diff
@@ -262,12 +263,12 @@ sub supports_image_arch {
     (grep { $_ eq $arch } @{$images_uri{$distri}{$version}{available_arch}}) ? 1 : 0;
 }
 
-# Returns a tuple of image urls.
+# Returns a tuple of single image url.
```

Comment:
```suggestion
# Returns an array with the contents of the given variable or undef, if the var is not set
```
Sorry, I'm nitpicking here :slightly_smiling_face: 

AFAIK perl has no tuples, also it does not return empty but undef if the var is not set. I made a suggestion how this can be addressed.

Reply:
Nice, thank you :smile:

Reply:
more specifically a `ref to array`. one question. @pdostal  Why dont you use perlpod format?